framework/auth/oauth_scopes.py

```diff
@@ -71,6 +77,9 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ,)
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
 
+    USERS_FULL_READ = USERS_READ + (CoreScopes.USERS_ADDON_READ,)
```

Comment:
Other than that, if there's a use case for third parties, that makes sense to not lock it down completely; I'm just encouraging us to think about the boundaries around data as we go, so that it will be easier to release more restrictive scopes later when we need them.